vault/resource_github_auth_backend.go

```diff
@@ -167,6 +172,13 @@ func githubAuthBackendRead(d *schema.ResourceData, meta interface{}) error {
 	path := "auth/" + d.Id()
 	configPath := path + "/config"
 
+	log.Printf("[DEBUG] Reading github auth mount from '%q'", path)
+	mount, err := authMountInfoGet(client, d.Id())
+	if err != nil {
+		return fmt.Errorf("error reading github auth mount from '%q': %s", path, err)
+	}
+	log.Printf("[INFO] Read github auth mount from '%q'", path)
```

Comment:
Minor thing I learned recently - if you use `%q` it automatically adds quotes surrounding the field. [Check this out.](https://play.golang.org/p/fmEhA2257Wi) This is in no way a blocker for the PR, just an interesting tidbit.

Reply:
Thanks, Becca. Really nice tip to know!